packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -292,6 +295,11 @@ export interface IColumn {
    * An optional class name to stick on the column cell within each header.
    */
   headerClassName?: string;
+
+  /**
+   * If set, will add LTR margin-right to column and cells.
```

Comment:
Adds margin-right "padding" for detail headers and detail row cells.  Included screenshot with border: 1px solid red on cell headers

![screen shot 2017-05-31 at 4 00 05 pm](https://cloud.githubusercontent.com/assets/1291968/26657706/5f84cf26-461a-11e7-805a-9242b66f3baf.png)